lib/universal/mapper.js

```diff
@@ -579,6 +579,8 @@ function createCommonGAForm(facade, settings) {
   var app = facade.proxy('context.app') || {};
   var traits = facade.traits();
   var options = facade.options('Google Analytics');
+  var os = facade.proxy('context.os');
```

Comment:
`os` is fine, since the only two values there are `name` and `version`, which I require to mimic `userAgent`. I will change the device info there though.